src/migrateCalculatedMeasure.ts

```diff
@@ -11,18 +17,50 @@ interface LegacyCalculatedMeasure {
 export function migrateCalculatedMeasure(
```

Comment:
I still don't get why we have this indirection: as I suggested in [this snippet](https://github.com/activeviam/activeui-migration/pull/26#discussion_r932062876), I would rather inline this in the `forEach` directly in `migrateCalculatedMeasures` , because:
- it's simpler code-split wise
- it allows to do everything in one loop, rather having two first generate all the intermediate results, then go through them all

Reply:
Ahh I see, I've just added a commit that inlines this directly in `migrateCalculatedMeasures`